docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
```

Comment:
Since we're using `docker-compose` here, what's your opinion on moving all the logic in here (and skip the `Dockerfile` and `build` step) with something like the "if you prefer Docker Compose" section of this: https://github.com/nodejs/docker-node#create-a-dockerfile-in-your-nodejs-app-project

I think it could simplify the overhead and complexity a bit.

Reply:
The build step is nice because we build in a single place, then upload that image to a registry which is later downloaded for running tasks in parallel across many machines.

My recommendation is a big Dockerfile and a small docker-compose file. It's nice to be able to run docker commands manually without docker-compose occasionally. Maybe I'll explore doing this as well though.

Reply:
> My recommendation is a big Dockerfile and a small docker-compose file.

In general I think my instinct on this would be to agree with you on that. I was sort of drawn to the alternative because of the simplicity of this particular repo/use caseI think. 

Might be worth sticking with that principal regardless though for the structure and consistency.